src/metabase/lib/filter/update.cljc

```diff
@@ -217,8 +237,13 @@
     longitude-column                             :- :some
     {:keys [north east south west], :as _bounds} :- [:ref ::lat-lon.bounds]]
    (-> query
-       (remove-existing-filters-against-column stage-number latitude-column)
-       (remove-existing-filters-against-column stage-number longitude-column)
-       (lib.filter/filter stage-number (let [[lat-min lat-max] (sort [north south])
-                                             [lon-min lon-max] (sort [east west])]
```

Comment:
Worth calling out that this func used to sort the bounds, which is not compatible with receiving wrapped bounds where `east` might be < than `west` for bounds that cross the 180th meridian.